src/actions/posts.ts

```diff
@@ -381,17 +380,21 @@ export function deletePost(post: ExtendedPost) {
                 }
             });
         } else {
-            dispatch({
-                type: PostTypes.POST_DELETED,
-                data: delPost,
-                meta: {
-                    offline: {
-                        effect: () => Client4.deletePost(post.id),
-                        commit: {type: 'do_nothing'}, // redux-offline always needs to dispatch something on commit
-                        rollback: receivedPost(delPost),
-                    },
-                },
-            });
+            try {
+                dispatch({
+                    type: PostTypes.POST_DELETED,
+                    data: delPost,
+                });
+
+                await Client4.deletePost(post.id);
+            } catch (e) {
+                // Recovering from this state doesn't actually work. The deleteAndRemovePost action
```

Comment:
I played with this for a while too: calling this comment out.

Reply:
Is this a follow-up ticket to have the delete modal report back to the user? Or maybe pop a toastr or something to say the post was never deleted?

Reply:
Good call: filed https://mattermost.atlassian.net/browse/MM-33634 to track.